shopfloor_packing_info/views/shopfloor_packing_info_views.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="shopfloor_packing_info_tree_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info tree</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="text" />
+                <field name="active" />
+            </tree>
+        </field>
+    </record>
+    <record id="shopfloor_packing_info_form_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info form</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="active" invisible="1" />
+                    <button
+                        type="object"
+                        name="toggle_active"
+                        string="Archive"
+                        attrs="{'invisible': [('active', '=', False)]}"
+                    />
+                    <button
+                        type="object"
+                        name="toggle_active"
+                        string="Restore"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                            <field name="text" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="shopfloor_packing_info_search_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info search</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name" />
+                <field name="text" />
+                <field name="active" />
+                <separator />
+                <filter
+                    string="All"
+                    name="all"
+                    domain="[('active', 'in', [False, True])]"
+                />
+            </search>
+        </field>
+    </record>
+    <record id="action_shopfloor_packing_info" model="ir.actions.act_window">
+        <field name="name">Checkout Packing Informations</field>
+        <field name="res_model">shopfloor.packing.info</field>
+        <field name="view_mode">tree,form</field>
+        <field name="context">{}</field>
+        <field
+            name="view_ids"
+            eval="[(5, 0, 0),
+            (0, 0, {'view_mode': 'tree',
+                    'view_id': ref('shopfloor_packing_info.shopfloor_packing_info_tree_view')}),
+            (0, 0, {'view_mode': 'form',
+                    'view_id': ref('shopfloor_packing_info.shopfloor_packing_info_form_view')})]"
```

Comment:
you shouldn't need to declare these if you have only one view of each and they'll be the default ones

Reply:
That's the thing: right now there is just one view. By indicating the view we want, we protect against future views. This was done on purpose and I like to do it always this way (sometimes I use an explicit record of ```ir.actions.act_window.view``` for this... )

Reply:
Generally you should never need this because the views you reference have a priority of 16, and when we add views that we don't want to be picked by default, we put a greater number such as 99. If you look in official or OCA addon, you'll never see this for default views, unless the views have a priority which would not make them the default.